timemachine/cpp/src/kernels/k_barostat.cuh

```diff
@@ -128,7 +130,7 @@ void __global__ k_decide_move(
     const bool rejected = w > 0 && rand[1] > std::exp(-w / kt);
 
     while (idx < N) {
-        if (idx == 0) {
+        if (idx == 0 && adaptive) {
```

Comment:
nit: this seems like it also disables accumulation of accept/reject counts when `adaptive` is false? Is there a benefit to maintaining the stats? (e.g. would they be useful for diagnostics, or to better inform moves the next time `adaptive` is enabled)

Reply:
+1 for maintaining acceptance/trial counters even when not currently being used to adapt proposal scale.

Reply:
Changed to maintain counter in [7d71499](https://github.com/proteneer/timemachine/pull/1152/commits/7d71499b1b972c6c8f45f6b2726b4901acdd84d0)